pkgs/development/interpreters/python/tests.nix

```diff
@@ -50,6 +51,14 @@ let
     # };
   };
 
+  integrationTests = lib.recurseIntoAttrs {
+    # Before the addition of NIX_PYTHONPREFIX mypy was broken
+    nix-pythonprefix-mypy = import ./tests/test_nix_pythonprefix {
```

Comment:
Every interpreter has this set of tests. Therefore, don't use `pkgs.python3`. If it only works with `python3`, use something like `python.isPy3k` to restrict.

Reply:
Done.